python/packages/nisar/workflows/focus.py

```diff
@@ -1101,7 +1101,7 @@ def process_rfi(cfg: Struct, raw_data: np.ndarray,
             num_max_trim=opt_evd.num_max_trim,
             num_min_trim=opt_evd.num_min_trim,
             max_num_rfi_ev=opt_evd.max_num_rfi_ev,
-            num_rng_blks=opt.num_range_blocks,
+            num_samples_rng_blk=opt.num_samples_rng_blk,
```

Comment:
Hmm, I don't think these changes are quite ready for review yet. Please test your changes to `focus.py` with both the ST-EVD algorithm and the FDNF algorithm. I don't see how either could be run without errors in the current state.

```suggestion
            num_samples_rng_blk=opt_evd.num_samples_rng_blk,
```